File: Python/Iniciante/1050.py
# Problema 1050 - Beecrowd - Iniciante - Nível 2

# Recebe o ddd
ddd = int(input())

# Estado inicial da existencia do ddd no dicionário
existe = False

# Declaração do dicionário com os ddd's e seus respectivos estados
cidades = {
        61: "Brasilia", 71: "Salvador",
        11: "Sao Paulo", 21: "Rio de Janeiro",
        32: "Juiz de Fora", 19: "Campinas",
        27: "Vitoria", 31: "Belo Horizonte"
        }

# Itera sobre o dicionário em busca do ddd informado
for i in cidades:
    if i == ddd:
        print(cidades[i])
        # Indica que encontrou o ddd na lista
        existe = True
        break

if not(existe):
    print("DDD nao cadastrado")

File: Rust/Iniciante/1050.rs
use std::io;
use std::error::Error;
use std::collections::HashMap;

fn main() -> Result<(), Box<dyn Error>> {
	let stdin = io::stdin();
	let mut buffer = String::new();

	stdin.read_line(&mut buffer)?;

	let mut tabela = HashMap::new();
	tabela.insert(61, "Brasilia");
	tabela.insert(71, "Salvador");
	tabela.insert(11, "Sao Paulo");
	tabela.insert(21, "Rio de Janeiro");
	tabela.insert(32, "Juiz de Fora");
	tabela.insert(19, "Campinas");
	tabela.insert(27, "Vitoria");
	tabela.insert(31, "Belo Horizonte");

	let ddd: i64 = buffer.trim().parse()?;

	if tabela.contains_key(&ddd) {
		println!("{}", tabela.get(&ddd).unwrap());
	} else {
		println!("DDD nao cadastrado");
	}

	Ok(())
}